js/index.js

```diff
@@ -0,0 +1,19 @@
+const today = new Date();
+console.log(today); //Sun Nov 13 2022 07:59:05 GMT-0800 (Pacific Standard Time)
+const thisYear = today.getFullYear();
+console.log(thisYear); //2022
+const footer = document.querySelector("footer");
+const copyright = document.createElement("p");
+let mName = "Abebe";
+copyright.innerHTML = `${mName} ${thisYear}`;
+footer.appendChild(copyright);
+const skills = ["HTML", "CSS", "JavaScript", "Cooking"];
+// console.log(skills);
+ const skillSection = document.getElementById("skills"); // Do I need this line?
+const skillsList = document.querySelector("ul");
```

Comment:
```suggestion
const skillsList = skillSection.querySelector("ul");
```

Instead of using `querySelector` on the entire document, you can have it just search within a particular element, in this case `skillSection`, which means less work to have to search for the element.